wiki/Client/Beatmap_editor/AiMod/en.md

```diff
@@ -25,139 +25,143 @@ Development of AiMod usually lags behind the standards held by the mapping commu
+| These two objects exist at the same point in time! | Two objects are placed exactly on top of each other timeline-wise, requiring the player to click both objects simultaneously. | Find the two objects and either move or delete one of them. | |
+| There are no hitsounds on any objects. Hitsounds are required for rankable beatmaps. | *Self-explanatory* | Hitsound the map using whistles, claps and finishes. | |
+| This spinner appears onscreen later than objects which follow it. | Objects appear while the spinner is active. This could be caused by having low AR and/or placing objects too close (timeline-wise) to the spinner. | Change the length of the spinner and/or remove the following objects. | |
+| This hold note is less than 10ms long! | A hold note is presented at an unreasonably short length, rendering the map impossible to SS. | Increase the length of the hold note or remove it. | ![](/wiki/shared/mode/mania.png) osu!mania maps only |
+| This object overlaps with another object. | Two objects are overlapped by each other on the same column in time, rendering the map impossible to SS. | Remove one of the overlapping objects. | ![](/wiki/shared/mode/mania.png) osu!mania maps only |
+| This object is stacked on top of another object. | Two objects are stacked on top of each other on the same column in time, rendering the map impossible to SS. | Find the stacked keys or hold notes and either move or delete one of them. | ![](/wiki/shared/mode/mania.png) osu!mania maps only |
+| More than 6 notes simultaneously is not allowed. | One or more patterns in the map requires the player to actively press more than 6 keys at a time. On certain cases, this is discouraged by the [osu!mania ranking criteria](/wiki/Ranking_Criteria/osu!mania) as most keyboards can only allow up to 6 key presses at once due to a phenomenon known as [ghosting](https://nelson-miller.com/what-is-keyboard-ghosting-and-how-do-you-prevent-it/). | Check your beatmap to make sure that each column has at most 6 simultaneous note presses. | ![](/wiki/shared/mode/mania.png) osu!mania maps only |
 
 #### Design
 
-| Message | Explanation | Solution |
-| :-- | :-- | :-- |
-| Your beatmap has no background image. | The current difficulty doesn't have a background image! Note that videos don't count as background images because the user isn't required to download them. | Find a suitable image and use it as the background image. |
-| File missing:{0} | Your beatmap is missing files. | Recover the missing files or make sure the beatmap is not trying to use non-existent files. |
+| Message | Explanation | Solution | Notes |
+| :-- | :-- | :-- |  :-- |
+| Your beatmap has no background image. | *Self-explanatory* | Find a suitable image and use it as the background image. | Videos don't count as background images because the user isn't required to download them. |
```

Comment:
contractions shouldn't be used in wiki articles

```suggestion
| Your beatmap has no background image. | *Self-explanatory* | Find a suitable image and use it as the background image. | Videos do not count as background images because the user is not required to download them. |
```